docs/DeveloperGuide.md

```diff
@@ -1355,6 +1427,64 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case 8: Edit the score(s) of a graded test**
+
+**MSS**
+
+1.  User requests to list persons.
+2.  F.A.K.E.J.A.R.V.I.S. shows a list of persons.
+3.  User requests to edit the score(s) of a graded test for a specific person in the list.
+4.  F.A.K.E.J.A.R.V.I.S. edits score(s) of the person's graded test.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+  * 3a1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case resumes at step 2.
+
+* 3b. The parameter(s) given is/are invalid.
+
+  * 3b1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case resumes at step 2.
+
+* 3c. Extra parameter(s) is/are given (e.g `editgradedtest 1 ra1/<SCORE> ra2/<SCORE> mt/<SCORE> f/<SCORE> pe/<SCORE> ra1/<SCORE>`)
+
+  * 3c1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case resumes at step 2.
+
+* 3d. Fewer parameter(s) is/are given (e.g `editgradedtest 1 ra1/<SCORE> ra2/<SCORE>`)
+
+  * 3d1. F.A.K.E.J.A.R.V.I.S. shows an success message.
+
+    Use case resumes at step 4.
+
+* 3e. The parameter(s) are in different order (e.g `editgradedtest 1 ra1/<SCORE> pe/<SCORE> f/<SCORE> ra2/<SCORE> mt/<SCORE>`)
+
+  * 3e1. F.A.K.E.J.A.R.V.I.S. shows an success message.
+
+    Use case resumes at step 4.
+
+* 3f. The given score is invalid.
+
+  * 3f1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case resumes at step 2.
+
+* 3g. The given score is invalid.
+
+  * 3g1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case resumes at step 2.
 
 
 *{More to be added}*
```

Comment:
Just a nit, but think we could remove this already

Reply:
ok, will delete that :)